cmd/kube-controller-manager/app/names/controller_names.go

```diff
@@ -0,0 +1,137 @@
+//     3.2. when defined flag's help mentions a controller name
+//  4. defining a new service account for a new controller (old controllers may have inconsistent service accounts to stay backwards compatible)
+const (
+	ServiceAccountTokenController                = "serviceaccount-token-controller"
+	EndpointController                           = "endpoint-controller"
+	EndpointSliceController                      = "endpointslice-controller"
+	EndpointSliceMirroringController             = "endpointslice-mirroring-controller"
+	ReplicationControllerController              = "replicationcontroller-controller"
+	PodGarbageCollectorController                = "pod-garbage-collector-controller"
+	ResourceQuotaController                      = "resourcequota-controller"
+	NamespaceController                          = "namespace-controller"
+	ServiceAccountController                     = "serviceaccount-controller"
+	GarbageCollectorController                   = "garbage-collector-controller"
+	DaemonSetController                          = "daemonset-controller"
+	JobController                                = "job-controller"
+	DeploymentController                         = "deployment-controller"
+	ReplicaSetController                         = "replicaset-controller"
+	HorizontalPodAutoscalerController            = "horizontal-pod-autoscaler-controller"
+	DisruptionController                         = "disruption-controller"
+	StatefulSetController                        = "statefulset-controller"
+	CronJobController                            = "cronjob-controller"
+	CertificateSigningRequestSigningController   = "certificatesigningrequest-signing-controller"
+	CertificateSigningRequestApprovingController = "certificatesigningrequest-approving-controller"
+	CertificateSigningRequestCleanerController   = "certificatesigningrequest-cleaner-controller"
+	TTLController                                = "ttl-controller"
+	BootstrapSignerController                    = "bootstrap-signer-controller"
+	TokenCleanerController                       = "token-cleaner-controller"
+	NodeIpamController                           = "node-ipam-controller"
+	NodeLifecycleController                      = "node-lifecycle-controller"
+	PersistentVolumeBinderController             = "persistentvolume-binder-controller"
+	AttachDetachController                       = "attach-detach-controller"
+	PersistentVolumeExpanderController           = "persistentvolume-expander-controller"
+	ClusterRoleAggregationController             = "clusterrole-aggregation-controller"
+	PersistentVolumeClaimProtectionController    = "persistentvolumeclaim-protection-controller"
+	PersistentVolumeProtectionController         = "persistentvolume-protection-controller"
+	TTLAfterFinishedController                   = "ttl-after-finished-controller"
```

Comment:
pod-ttl-after... ?

Reply:
not sure here. Now it primarly operates on jobs (transitively on pods), but the KEP mentions this:

```
1. We also need the TTL controller to clean up finished Pods in the future, and we consider
   generalizing TTL controller later for custom resources. 
```

so probably better to leave it be?

Reply:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/592-ttl-after-finish/README.md#ttl-controller

Reply:
OK